PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/insights/ApplicationInsightsComboBox.java

```diff
@@ -50,13 +54,30 @@ public void setSubscription(Subscription subscription) {
         this.refreshItems();
     }
 
+    public ApplicationInsightsConfig getNewInsightsConfig() {
```

Comment:
why not use getValue?

Reply:
This is an outdated method for auto created insights, have fixed in later commit